assets/components/countdown/countdown.jsx

```diff
@@ -71,17 +78,29 @@ export default class Countdown extends Component<PropTypes, StateTypes> {
       days, hours, minutes, seconds,
     } = this.state.time;
 
-    const units = days > 0 ? [days, hours, minutes, seconds] : [hours, minutes, seconds];
+    const { legend } = this.props;
+
+    const chip = (description: string, time: number) => (
+      <span className="component-countdown__chip">
+        <span className="component-countdown__time">{addLeadingZeros(nonNegative((time)), 2)}</span>
+        <span className="component-countdown__description">{description}</span>
+      </span>
+    );
 
     return (
       <time className="component-countdown">
-        {units.map((unit, index) => (
-          <span>
-            <span className="component-countdown__time">{addLeadingZeros(unit, 2)}</span>
-            {index < units.length - 1 && ':'}
+        {days > 0 && chip('days', days)}
+        {chip('hrs', hours)}
+        {chip('mins', minutes)}
+        {chip('secs', seconds)}
```

Comment:
In terms of a11y, I checked the [GDS guidance](https://accessibility.blog.gov.uk/2017/02/08/advice-for-creating-content-that-works-well-with-screen-readers/) and they actually encourage _against_ using any type of additional markup for this, just roll with the awkward spellings.